config/settings/stupoli.py

```diff
@@ -0,0 +1,18 @@
+# Custom settings for stupoli deploy
+# -----------------------------------------------------------------------------
+OSLER_DISPLAY_REFERRALS = False
+OSLER_DISPLAY_APPOINTMENTS = False
+OSLER_DISPLAY_CASE_MANAGERS = False
+OSLER_DISPLAY_ATTESTABLE_BASIC_NOTE = False
+OSLER_DISPLAY_DIAGNOSIS = False
+OSLER_DISPLAY_VOUCHERS = False
+OSLER_DISPLAY_WILL_RETURN = False
+OSLER_DISPLAY_ATTENDANCE = True
+
+OSLER_DEFAULT_CITY = "Kansas City"
+OSLER_DEFAULT_STATE = "MO"
+OSLER_DEFAULT_ZIP_CODE = "64106"
+OSLER_DEFAULT_COUNTRY = "USA"
+OSLER_DEFAULT_ADDRESS = "205 East 9th St."
+OSLER_HOMELESS_OPTION = True
+OSLER_PHONE_NUMBER_OPTION = False
```

Comment:
We should probably set these defaults correctly or not at all.

Reply:
What do you mean by setting them correctly?